binding.gyp

```diff
@@ -28,8 +28,7 @@
         './src/shadow_node.cpp',
       ],
       'include_dirs': [
-        '.',
-        "<!(node -e \"require('nan')\")",
+        '<!@(node -p "require(\"node-addon-api\").include")',
```

Comment:
I notice that a consistent failure on Actions https://github.com/RobotWebTools/rclnodejs/actions/runs/13857494263/job/38777515813?pr=1088#step:7:25, can you pass the compiling locally?